src/core/server/app/router/client.ts

```diff
@@ -265,6 +272,7 @@ export async function mountClientRoutes(
       cacheDuration: false,
       disableFraming: true,
       entrypointLoader: manifestLoader.createEntrypointLoader("auth"),
+      templateVariables: { title: "Sign in" },
```

Comment:
Are we making this a configurable item via the env variables/config?

From what I see, this is going to override the title with this "Sign in" default. Do we want to do a config via an env var?

If we don't do an env config, do we want to translate this? I don't know if this handler is initialized too early to use i18n and the tenant locale or not, but it would be nice if this title could be localized.

Thoughts?

Reply:
I think translation is the way to go, and these templates should be passed through translation. Looking into that.

Reply:
Scratch that, translation wasn't feasible, so I've added an env var